### PR TITLE
Added landing page, header and navbar-footer

### PR DESCRIPTION
**Pull request by OliverGraml**
I would really appreciate if you could check my following work: 
- Updated my landing page with a background image
- Added a header with a logo-icon and a home-button
- Created a NavBar for my landing page as a circle cloud

### CODE REVIEW

**Comment by GabrieleGrimme** [comment]
Looks good to me. Very well coded (with all the Einrückungen ;-)

File: client/src/components/NavFooterLP.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+      </NavLink>
+    </NavWrapper>
+  );
+}
+
+const NavWrapper = styled.footer`
+  display: grid;
+  grid-template-columns: 144px 144px;
+  grid-template-rows: 144px 144px;
+  justify-content: center;
```
Comment (jensdreske) [inline client/src/components/NavFooterLP.js:27]: has no effect on the buttons, because the navlinks are in the way

Comment (jensdreske) [reply client/src/components/NavFooterLP.js:27]: - background image looks nice and responsive
- logo looks nice and back home button works
- i like the idea of the circle cloud

Comment (OliverGraml) [reply client/src/components/NavFooterLP.js:27]: Thank you very much for your review and the feedback, Jens.

**Pull request merged**